shotover-proxy/src/transforms/redis/sink_cluster.rs

```diff
@@ -482,7 +492,10 @@ impl RedisSinkCluster {
     #[inline(always)]
     fn short_circuit(&self, one_tx: oneshot::Sender<Response>) {
         warn!("Could not route request - short circuiting");
-        if let Err(e) = self.send_error_response(one_tx, "ERR Could not route request") {
+        if let Err(e) = self.send_error_response(
+            one_tx,
+            "ERR Shotover RedisSinkCluster does not not support this command used in this way",
```

Comment:
Are we able to give more detailed error message?

Reply:
What sort of details do you want?

Reply:
What way the command is being used

Reply:
ah, well we could try and store an error in the RoutingInfo::Unsupported but I think that should go in a follow up PR

Reply:
Happy for that to be attempted in a followup PR.